thinkhazard/templates/base.jinja2

```diff
@@ -29,6 +29,13 @@
       var app = {};
       app.administrativedivisionUrl = '{{'administrativedivision'|route_url}}';
       app.reportpageUrl = '{{'report'|route_url}}';
+      app.mapImgUrl = '{{'map'|route_url(format='png')}}';
```

Comment:
did you try jpeg instead ? 

It might not look as good, but loading time could be improved.